contrib/resources/translations/en.lng

```diff
@@ -700,7 +700,7 @@ Writing config file %s
 
 .
 :SHELL_CMD_CONFIG_HELP_LONG
-Config tool:
+Adjusts DOSBox Staging's configurable values.
```

Comment:
Would "configuration parameters" sound a bit better and more specific?

Reply:
Or maybe just "DOSBox Staging configuration tool."?

Reply:
Might as well as a third comment! 

The `config` tool specifically changes `conf file` settings in realtime (as opposed to settings that come in via the command line arguments or environment variables), so my suggestion would be to incorporate that aspect.

@Wengier, feel free to meld/adjust (or ignore all of them!) and merge when you're ready :+1:

Reply:
@Wengier mentioned: "the config tool does this and that, but I think its core function is to adjust staging’s configurable parameters.", so will merge as-is.

Reply:
I changed the word “values” to “parameters” as suggested right before @kcgen merged it.